src/data/services/DiscoveryDataApiService.js

```diff
@@ -37,8 +37,15 @@ class DiscoveryDataApiService {
   }
 
   static fetchOrganizations() {
+    const queryParams = {
+      page: 1,
+      page_size: 500,
+      limit: 500,
+    };
```

Comment:
This assumes we only have 500 orgs... 

When is this used? Can we add a flag that's like - show me only my orgs?